source/topics/paas/how-to-set-up-codeship.md

```diff
@@ -0,0 +1,11 @@
+To deploy from Codeship on Aptible, we recommend the following approach:
+
+1. Create a “Robots” role in your Aptible organization, and grant it “Read” and “Manage” permissions on your production account.
+2. Invite a new deploy user to this Robots role. It needs to have a real email address, but can be something like deploy@yourdomain.com.
+3. Accept the invitation for deploy@yourdomain.com, activate the account, and upload the public key from your projects _General_ settings page on your Aptible User Settings page.
```

Comment:
Can we clarify this a bit? The "General" settings for the project are in Codeship, correct? (Also, minor: "project's".)